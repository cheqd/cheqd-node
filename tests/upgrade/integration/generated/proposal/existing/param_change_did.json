{
    "title": "Did Param Change",
    "description": "This is a simple example of a param change - double the values",
    "type": "ParameterChange",
    "changes": [
        {
            "subspace": "cheqd",
            "key": "feeparams",
            "value": {"create_did": {"denom": "ncheq", "amount": "100000000000"}, "update_did": {"denom": "ncheq", "amount": "50000000000"}, "deactivate_did": {"denom": "ncheq", "amount": "20000000000"}, "burn_factor": "0.600000000000000000"}
        }
    ],
    "deposit": "10000000000ncheq"
}