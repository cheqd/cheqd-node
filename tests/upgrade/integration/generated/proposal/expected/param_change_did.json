{
    "create_did": {
        "denom": "ncheq",
        "amount": "100000000000"
    },
    "update_did": {
        "denom": "ncheq",
        "amount": "50000000000"
    },
    "deactivate_did": {
        "denom": "ncheq",
        "amount": "20000000000"
    },
    "burn_factor": "0.600000000000000000"
}